plugin/rpcplugin/sandbox/sandbox_linux.go

```diff
@@ -0,0 +1,468 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package sandbox
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"syscall"
+	"unsafe"
+
+	"github.com/pkg/errors"
+	"golang.org/x/sys/unix"
+
+	"github.com/mattermost/mattermost-server/plugin/rpcplugin"
+)
+
+func init() {
+	if len(os.Args) < 3 || os.Args[0] != "sandbox.runProcess" {
+		return
+	}
+
+	var config Configuration
+	if err := json.Unmarshal([]byte(os.Args[1]), &config); err != nil {
+		fmt.Println(err.Error())
+		os.Exit(1)
+	}
+	if err := runProcess(&config, os.Args[2]); err != nil {
+		if eerr, ok := err.(*exec.ExitError); ok {
+			if status, ok := eerr.Sys().(syscall.WaitStatus); ok {
+				os.Exit(status.ExitStatus())
+			}
+		}
+		fmt.Println(err.Error())
+		os.Exit(1)
+	}
+	os.Exit(0)
+}
+
+func systemMountPoints() (points []*MountPoint) {
+	points = append(points, &MountPoint{
+		Source:      "proc",
```

Comment:
Should the source be `/proc` as well?

Reply:
For mount type "proc", the source isn't a file path. I'm not actually sure what it is or if it's even allowed to be anything other than "proc", but `mount -t proc proc /proc` is the canonical way to mount a proc filesystem (http://man7.org/linux/man-pages/man5/proc.5.html). You can also see that this is what Docker uses for the proc filesystem source: https://github.com/moby/moby/blob/d4239a6e286f546f5eda507437f643b5a1548c5d/oci/defaults.go#L74